doc/reduce-traffic.md

```diff
@@ -35,3 +35,14 @@ blocks and transactions to fewer nodes.
 Reducing the maximum connected nodes to a minimum could be desirable if traffic
 limits are tiny. Keep in mind that bitcoin's trustless model works best if you are
 connected to a handful of nodes.
+
+## 4. Turn off transaction relay (`-blocksonly`)
+
+Forwarding transactions to peers increases the P2P traffic. To only sync blocks
+with other peers, you can disable transaction relay.
+
+Be reminded that this setting could hurt your privacy if used while a wallet is
+loaded or if you use the node to broadcast transactions.
+
+Moreover, this makes block propagation slower because compact block relay can
+only be used when transaction relay is enabled
```

Comment:
Suggest that you also document that using `-blocksonly` stops fee estimation from working.

nit: end sentence with a `.`.

Reply:
Oof, good point!